<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>744</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Course Schedule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>1091</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Add new course</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Code of the Course</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Name of the Course</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Name of the Instructor</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Credit_Combo">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Credit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Elective or Compulsary</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Department or Service</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TA_code_new_Course">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TA_name_new_Course">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TA_Instructor_new_Course">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Department_Combo">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>50</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>S</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="BT_Add_new_course_">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Course</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_year">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>50</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_status">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>50</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>1101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>Busy Courses</string>
    </property>
    <widget class="QComboBox" name="Busy_Courses_comboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="busy_days_combo">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Monday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wednesday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thursday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Friday</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="BT_Add_Busy_course_">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Add Busy Time</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_time">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Morning</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Afternoon</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_bring">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Bring Courses</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BT_CreateSchedule">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Create Schedule</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>1101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Classrooms</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Number of Big Clasrooms:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TA_Big_Class">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Number of Small Clasrooms:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TA_Small_Class">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="button_addClasses">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Classes</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="title">
     <string>Remove a Course</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Remove Choosen Course</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Courses:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_REmove">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>CENG104</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ENG102</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG202</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG204</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG206</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG302</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG304</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG306</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG310</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG316</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG317</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG318</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG415</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ENGR404</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG404</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG427</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG451</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG466</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG470</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG472</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CENG474</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ENGR402</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>871</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
